kpatch-build/kpatch-gcc

```diff
@@ -15,7 +15,10 @@ if [[ "$TOOLCHAINCMD" = "gcc" ]] ; then
 	while [ "$#" -gt 0 ]; do
 		if [ "$1" = "-o" ]; then
 			obj=$2
-			[[ $2 = */.tmp_*.o ]] && obj=${2/.tmp_/}
+			# skip copying the original .o files, when .tmp_mc_*.o
+			# is passed from recordmcount.pl.
+			[[ $2 = */.tmp_mc_*.o ]] && break;
+			[[ $obj = */.tmp_*.o ]] && obj=${2/.tmp_/}
```

Comment:
The 1st line seems to match the comment, but what's the purpose of the 2nd line?

Also it would be clearer to use the $obj variable everywhere instead of $2.